build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml

```diff
@@ -84,6 +84,49 @@ jobs:
       artifactName: 'mrtcore_test_binaries_$(buildPlatform)'
       downloadPath: $(testPayloadDir)
 
+  - script: |
+      echo source: $(Build.SourcesDirectory)\BuildOutput\Release\$(buildPlatform)
+      echo dest: $(Agent.TempDirectory)\pushnotification_testbin\$(buildPlatform)
+      dir $(Build.SourcesDirectory)\BuildOutput\Release\$(buildPlatform)
+      dir D:\
+      dir D:\a\
+      dir D:\a\1\
+      dir D:\a\1\s\
+      dir D:\a\1\s\BuildOutput\
+      dir D:\a\1\s\BuildOutput\Release\
+      dir D:\a\1\s\BuildOutput\Release\x64
+      dir D:\a\1\s\BuildOutput\Release\x64\PushNotificationTests
+    displayName: 'Copy push notification tests debug'
+
+  # Copy and Publish the test binaries so we can run them in Helix later.
+  - task: CopyFiles@2
+    displayName: 'copy push notifications test binaries'
+    condition: and(succeeded(), or(eq(variables['buildPlatform'], 'x86'), eq(variables['buildPlatform'], 'x64')))
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)\BuildOutput\Release\$(buildPlatform)'
+      Contents: |
+        PushNotificationTests\**
```

Comment:
I thought Helix infra was copying the entire BuildOutput/configuration/platform directory tree to the helix test environment. That's what the aggregator pipeline does for its testing

Reply:
@jefgen @alexlamtest